hardware/smart_utils_info.py

```diff
@@ -0,0 +1,27 @@
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+NVME_INFOS = {
```

Comment:
nit: I'd not include the colon in keys

Reply:
I would not too, although how the parser is built right now it will not work
I was thinking to follow up on this anyway, or I can apply the change here